exercises/meetup/canonical-data.json

```diff
@@ -3,446 +3,567 @@
-    },
-    {
-      "description": "second Wednesday of July 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 7,
-      "week": "second",
-      "dayofweek": "Wednesday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Wednesday of August 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 8,
-      "week": "second",
-      "dayofweek": "Wednesday",
-      "dayofmonth": 14
-    },
-    {
-      "description": "second Thursday of September 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 9,
-      "week": "second",
-      "dayofweek": "Thursday",
-      "dayofmonth": 12
-    },
-    {
-      "description": "second Thursday of October 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 10,
-      "week": "second",
-      "dayofweek": "Thursday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Friday of November 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 11,
-      "week": "second",
-      "dayofweek": "Friday",
-      "dayofmonth": 8
-    },
-    {
-      "description": "second Friday of December 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 12,
-      "week": "second",
-      "dayofweek": "Friday",
-      "dayofmonth": 13
-    },
-    {
-      "description": "second Saturday of January 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 1,
-      "week": "second",
-      "dayofweek": "Saturday",
-      "dayofmonth": 12
-    },
-    {
-      "description": "second Saturday of February 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 2,
-      "week": "second",
-      "dayofweek": "Saturday",
-      "dayofmonth": 9
-    },
-    {
-      "description": "second Sunday of March 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 3,
-      "week": "second",
-      "dayofweek": "Sunday",
-      "dayofmonth": 10
-    },
-    {
-      "description": "second Sunday of April 2013",
-      "property": "meetup",
-      "year": 2013,
-      "month": 4,
-      "week": "second",
-      "dayofweek": "Sunday",
-      "dayofmonth": 14
-    },
+      "description": "test -teenth",
+      "cases":[
+        {
+          "description": "monteenth",
+          "cases": [
+            {
+              "description": "testcase 1 : May 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 5,
+              "week": "teenth",
+              "dayofweek": "Monday",
+              "dayofmonth": 13
+            },
+            {
+              "description": "testcase 2 : August 2013",
+              "property": "meetup",
+              "year": 2013,
+              "month": 8,
+              "week": "teenth",
+              "dayofweek": "Monday",
+              "dayofmonth": 19
+            },
+            {
+              "description": "testcase 3 : September 2013",
```

Comment:
Why are there 3 tests?
What is different about them?
Why have we not tested June and July?